plugin/trino-ranger/README.md

```diff
@@ -0,0 +1,95 @@
+# Trino-Ranger plugin
+
+This plugin is designed to be build and run inside Trino.
+
+It works with vanilla Apache Ranger, version 2.2.1 and up. You dont need to customize Ranger at all.
+
+Here are the setup steps.
+
+
+### Full Enterprise ready Apache Ranger setup
+
+ * Install and startup Ranger
```

Comment:
I would like to break this up a bit and avoid the "quick and dirty" terminology. Quick may indicate that it should be simple and make someone feel dumb if they get stuck. Dirty may indicate that it's not ready for production. 

Happy to help you break this up. We'll also likely want to make some video enablement around this. That will give me a chance to actually run through and validate these steps and any potential hiccups users might encounter.

Reply:
> We'll also likely want to make some video enablement around this.

Ya, I was thinking the same thing. IMO, ASF Ranger community has some poor job of documentation. Worse I have seen.